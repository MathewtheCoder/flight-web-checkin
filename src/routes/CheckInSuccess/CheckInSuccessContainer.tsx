import React from 'react'

const CheckInSuccessContainer = () => {
    return (
        <div>
            Checkin Success
        </div>
    )
}

export default CheckInSuccessContainer