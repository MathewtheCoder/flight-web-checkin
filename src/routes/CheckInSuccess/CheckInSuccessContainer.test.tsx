import React from 'react';
import { shallow, toJson } from 'tests/enzyme';
import CheckInSuccessContainer from './CheckInSuccessContainer';

describe('CheckInSuccessContainer tests', () => {

  it('renders correctly CheckInSuccess page', () => {
    const wrapper = shallow(<CheckInSuccessContainer />);
    // Expect the wrapper object to be defined
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});