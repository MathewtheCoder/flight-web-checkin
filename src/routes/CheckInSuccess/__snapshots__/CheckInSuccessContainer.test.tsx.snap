// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckInSuccessContainer tests renders correctly CheckInSuccess page 1`] = `
<main
  className="checkinsuccess"
>
  <div
    className="wrapper"
  >
    <ForwardRef(CheckCircleFilled)
      className="check"
    />
    <h1>
      You've been checked in!
    </h1>
    <p>
      Enjoy your flight!
    </p>
    <Button
      block={true}
      className="actionBtn"
      htmlType="button"
      onClick={[Function]}
      size="large"
      type="primary"
    >
      CheckIn for another flight 
      <ForwardRef(RollbackOutlined) />
    </Button>
  </div>
</main>
`;

exports[`CheckInSuccessContainer tests renders list-items 1`] = `
<main
  className="checkinsuccess"
>
  <div
    className="wrapper"
  >
    <ForwardRef(CheckCircleFilled)
      className="check"
    />
    <h1>
      You've been checked in!
    </h1>
    <p>
      Enjoy your flight!
    </p>
    <Button
      block={true}
      className="actionBtn"
      htmlType="button"
      onClick={[Function]}
      size="large"
      type="primary"
    >
      CheckIn for another flight 
      <ForwardRef(RollbackOutlined) />
    </Button>
  </div>
</main>
`;
